### PR TITLE
[QOL] Expanded Admin Descriptions For ERT Deployment

### PR DESCRIPTION
**Pull request by Steelpoint**
This PR adds a quick extra description of what each level of the ERT deployment has in difference with one another. This is meant to help admins tell what the differences between each ERT deployment are.

The changed descriptions are:
- Blue: Light ERT (No Armoury Access)
- Amber: Full ERT (Armoury Access)
- Red: Elite ERT (Armoury Access + Pulse Guns)

### CODE REVIEW

File: code/modules/admin/verbs/one_click_antag.dm [hunk 1]
```diff
@@ -450,7 +450,7 @@
 
 // CENTCOM RESPONSE TEAM
 /datum/admins/proc/makeEmergencyresponseteam()
-	var/alert = input("Which team should we send?", "Select Response Level") as null|anything in list("Green: Centcom Official", "Blue: Light ERT", "Amber: Full ERT", "Red: Elite ERT", "Delta: Deathsquad")
+	var/alert = input("Which team should we send?", "Select Response Level") as null|anything in list("Green: Centcom Official", "Blue: Light ERT (No Guns)", "Amber: Full ERT (Energy Guns)", "Red: Elite ERT (Pulse Guns)", "Delta: Deathsquad")
```
Comment (MrPerson) [inline code/modules/admin/verbs/one_click_antag.dm]: Protip: You've just made the whole feature stop working

**Comment by Steelpoint** [comment]
It works now

**Comment by SconesC** [comment]
ert are still useless without red alert gear

**Comment by Steelpoint** [comment]
Blame that on Ikarrus, the red alert gear is how I originally envisioned the ERT.

But this really just gives admins some flexibility in sending in the ERT. It may not always be wanted to send in Pulse Guns to a situation where that might be too powerful.

**Comment by Ikarrus** [comment]
This is inaccurate. They still have energy guns on code blue

**Comment by Steelpoint** [comment]
Guess that goes to show since I've only seen red alert ERT's.

**Comment by Steelpoint** [comment]
Updated the text and the OP.

**Comment by Fayrik** [comment]
We could probably get rid of the code colours, they're fluff fair enough. But are admin buttons really the place for fluff?
If that information should be anywhere, it should be in the ERT spawn text.
Either way, Admin buttons should be descriptive of what they do, and this achieves that at the very least, so :+1:

**Pull request merged**